addons/website/static/tests/tours/edit_megamenu.js

```diff
@@ -120,6 +120,82 @@ registerWebsitePreviewTour('edit_megamenu', {
         trigger: ':iframe .o_mega_menu h4:contains("New Menu Item")',
     },
 ]);
+registerWebsitePreviewTour("megamenu_active_nav_link", {
+    url: "/",
+    edition: true,
+}, () => [
+       // Add a megamenu item to the top menu.
+    {
+        content: "Click on a menu item",
+        trigger: ":iframe .top_menu .nav-item a",
+        run: "click",
+    },
+    {
+        content: "Click on 'Link' to open Link Dialog",
+        trigger: ":iframe .o_edit_menu_popover a.js_edit_menu",
+        run: "click",
+    },
+    {
+        trigger: ".o_website_dialog",
+    },
+    {
+        content: "Trigger the link dialog (click 'Add Mega Menu Item')",
+        trigger: ".modal-body a:eq(1)",
+        run: "click",
+    },
+    {
+        content: "Write a label for the new menu item",
+        trigger: ".modal-dialog .o_website_dialog input",
+        run: "edit MegaTron",
+    },
+    {
+        content: "Confirm the mega menu label",
+        trigger: ".modal .modal-footer .btn-primary:contains(ok)",
+        run: "click",
+    },
+    {
+        trigger: `.oe_menu_editor [data-is-mega-menu="true"] .js_menu_label:contains("MegaTron")`,
+    },
+    {
+        content: "Save the website menu with a new mega menu",
+        trigger: ".modal-footer .btn-primary",
+        run: "click",
+    },
+    {
+        trigger: "#oe_snippets.o_loaded",
+    },
+    {
+        content: "Check for the new mega menu",
+        trigger: `:iframe .top_menu:has(.nav-item a.o_mega_menu_toggle:contains("Megatron"))`,
+    },
+    {
+        trigger: ".o_website_preview.editor_enable.editor_has_snippets:not(.o_is_blocked)"
+    },
+    clickOnExtraMenuItem({}, true),
+    toggleMegaMenu({}),
+    {
+        content: "Select the last menu link of the first column",
+        trigger: ":iframe .s_mega_menu_odoo_menu .row > div:first-child .nav > :nth-child(6)",
+        run: "click",
+    },
+    {
+        content: "Change the link",
+        trigger: "#o_link_dialog_url_input",
+        run: "edit /new_page"
```

Comment:
I think landing on a 404 page will make the runbot fail. Not sure...

Reply:
It's working on my local database... We'll see.